### DISCUSSION TITLE
How can I view the solution for a Battle

### DISCUSSION THREAD

**Discussion by Linh-Vo** [cssbattle/forum#341] [Q&A]
I solved the battle but I didn't know whether my solution is good or not. How can I see other player solutions or top solutions or something like that?

**Comment by Beowolve** [comment]
You can only see other solutions for "open" targets, like [#130](https://cssbattle.dev/play/130) for example. Solutions for other targets are sometimes shared on discord. But top solutions are usually kept as a secret ;).
